commander/README.md

```diff
@@ -16,7 +16,7 @@ Upon successful completion, NPM will add the `lisk-commander` executable `lisk`
 
 ## Usage
 
-```sh-session
+```shell
```

Comment:
I think this is auto generated

Reply:
by what? I just changed it, because it leaves a warning as TypeDoc doesn't know syntax highlighting for `sh-session`. 

But if this is not important, it can stay as-is.